metrics/density/README.md

```diff
@@ -7,7 +7,19 @@ This number of containers could be working under different configurations/condit
 - `docker_memory_usage.sh`: measures the Proportional Set Size (PSS) memory average
    for a number of containers launched in idle mode. This test uses the `sleep` command
    to allow any memory optimizations to 'settle' (e.g. KSM execution) for a configurable
-   period of time.
+   period of time. It also has an optional 'auto' mode which will try to determine when KSM
+   has settled down by examining the KSM `full_scans` and `pages_shared` values. auto mode
+   detection of KSM settling over-rides the timeout value for taking the measurements and
+   completing the test.
+- `docker_memory_usage.sh`: measures the Proportional Set Size (PSS) memory average
+   for a number of containers launched in idle mode. 
+   The test has two methods that can be utilized to wait for `KSM` to 'settle'.
+    - The test will sleep for the supplied `timeout` argument seconds before
```

Comment:
Lines 17-18 suggested rewrite: The test sleeps for the amount of time specified by 'timeout', which is in seconds, before taking measurements.